submissions/idiagheOsaigbovo/CssTask/Registration form/Regform.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>AfriDirect</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+
+    <h1>AfriDirect Registration Form</h1>
+
+
+    <div class="registration">
+               
+        <form id="registration" method="post" action="..php"></form>
+        
+        <h1>Registration</h1>
+        
+        <input type="text" name="Fname" id="name" placeholder="Enter firstname"><br>
+        <br>
+
+        <input type="text" name="Lname" id="name" placeholder="Enter lastname"><br>
+        <br>
+        <input type="text" name="Email" id="name" placeholder="Enter Email"><br>
+        <br>
+        <input type="text" name="Address" id="Address" placeholder="House Address"><br>
+        <br>
+        <input type="password" name="pass" id="name" placeholder="Enter password"> <br>
+        
+        <br>
+        <input type="password" name="pass" id="pass"  placeholder="Re-enter password"><br><br>
+
+        <select name="Number" id="num">
+            <option>+1</option>
+            <option>+44</option>
+            <option>+234</option>
+        </select> <input type="number" name="number" id="number" 
```

Comment:
format the tags here properly @osayiosas

Reply:
i just did  thank you